process/rewardTransaction/process.go

```
@@ -84,7 +84,7 @@ func (rtp *rewardTxProcessor) ProcessRewardTransaction(rTx *rewardTx.RewardTx) e
 	process.DisplayProcessTxDetails("ProcessRewardTransaction: receiver account details", accHandler, rTx)
 
 	err = accHandler.AddToBalance(rTx.Value)
```

this error is masked.

---

Added err check.